packages/db-dashboard/src/pages/ReactAdmin.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
```

Comment:
i've seen there are different colors, like a palette, but every time repeated, not only in this pull request, but in the whole dashboard package.

Can you add a color palette with the definition of the variables inside the root of the index.css file? @mcollina what do you think about it? or the css palette could be another issue?

Reply:
I think that this could be a good idea, but it could be another issue.
The issue https://github.com/platformatic/platformatic/issues/257 could be related to your comment.

PS
In this PR I have added the original css from React Admin with no changes.